opentok/opentok-meet
# Archive custom layout

#### What is this PR doing?
Adding a custom css for archiving.

#### How should this be manually tested?
With more than 3 participants in the room, start recording, start screen sharing, then stop screen sharing, and finally stop recording.
Check video recording: you should be able to see all the participants, then only the screen (screen sharing started), and then again the participants and no screen (screen sharing ended). Audio should be present the whole time.

#### What are the relevant tickets?
